src/python/pants/goal/run_tracker.py

```diff
@@ -191,6 +197,53 @@ def retrieve_logs(self) -> List[str]:
 
         return output
 
+    def _report(self) -> None:
+        total_run_time = "Unknown"
+        try:
+            total_run_time = str(self._total_run_time)
+        except:
+            pass
+
+        outcome = self._run_info.get("outcome", "Unknown")
+
+        content = textwrap.dedent(
+            """
+        ## Outcome
+        `{}`
+
+        ## Pants version
+        `{}`
+
+        ## Time
+        Timestamp: `{}`
+        Date: `{}`
+        Total run time: `{}`
+
+        ## Command
+        argv: `{}`
+        specs: `{}`
```

Comment:
IMO, we should remove both of these. This data should be anonymized, and this kind of information is very leaky.

Reply:
Yeah, something for us to finesse, I agree.